src/en/proposals/fairlysadpanda-revs-uplink.md

```diff
@@ -0,0 +1,159 @@
+
+The Terminal monitors the existence of the Head ID cards. Anyone can look at the terminal and know how many of the IDs are still "in play" - i.e. not destroyed or unreasonably far from the space station for recovery.
+
+In Revolutionary rounds, the Terminal can be exploited by a Head Revolutionary to initiate an unlinking sequence. This unlinking sequence works as follows:
+
+1. The Head Revolutionary gets at least one Head ID card.
+2. The Head Revolutionary breaks into the room with the terminal in it (probably the bridge).
+3. The Head Revolutionary\* inserts the ID card into the terminal.
+4. The Head Revolutionary waits through the timer.
+5. If the Head Revolutionary has any more Head ID cards, they can insert them to decrease the length of the timer.
+
+This timer blares out a warning that the station is being unlinked when the timer starts, with the time remaining. If the time remaining is modified (by inserting more ID cards), this is also announced.
+
+Any crew member can stop the countdown by interacting with the terminal and clicking a big obvious red flashing button on it. This stops the timer\*\*.
+
+If the timer reaches zero, the Revolutionaries win and a round restart is called. The Head Revolutionaries exploit the console to disable all security ~~ready for the Syndicate to show up~~ to free the crew from their corporate overlords and install themselves as "Consuls" of the station\*\*\*. Remaining living department heads and security officers are renamed "Enemies of the Revolution" and are ultra-valid, although given that the game is in end-of-round the murder has probably already started.
+
+_\*Revolutionaries can insert cards. It's also possible that they could activate the timer, but this needs playtesting._
+
+_\*\*The timer might reset to its starting value or might hold its starting value, so if restarted it continues counting down. This needs playtesting._
+
+_*Consul is the Roman term for an elected political leader - other names could be overseer, overlord, operative...*_
+
+# Head ID Cards
+
+Each time a department head spawns, they spawn with a special ID. This special ID has a big obvious chip on it that can be seen by examining the card. This chip is missing from "spare" IDs and not even the HoP can make new ones.
+
+The Head ID Cards are tracked by the Centcomm Uplink Terminal, so the state of the round can be known (is an ID inside the terminal, on the station, destroyed or out of range?).
+
+If every single Head ID Card has been removed via distance or destruction, the Uplink Terminal will notice and a _very annoyed_ CentComm will send a master override ID to the automated trade station. This Uplink Override ID counts as the Captain's ID.
+
+The following department heads have a Head ID Card:
+
+- The Captain
+- The Head of Personnel
+- The Head of Security
+- The Research Director
+- The Chief Medical Officer
+- The Quartermaster
```

Comment:
I assume the fact that the Chief Engineer is missing here is an oversight?

Reply:
Nah fuck engineering tbh

(yes it is, good spot)